src/BuildScriptGenerator/PlatformInstallerBase.cs

```diff
@@ -71,10 +72,16 @@ public abstract class PlatformInstallerBase
                 .AppendLine($"echo \"$checksumValue {version}.tar.gz\" | sha512sum -c - >/dev/null 2>&1")
                 .AppendLine("echo Extracting contents...")
                 .AppendLine($"tar -xzf {tarFile} -C .")
-                .AppendLine($"rm -f {tarFile}")
+                .AppendLine($"rm -f {tarFile}")             
                 .AppendLine("PLATFORM_SETUP_ELAPSED_TIME=$(($SECONDS - $PLATFORM_SETUP_START))")
                 .AppendLine("echo \"Done in $PLATFORM_SETUP_ELAPSED_TIME sec(s).\"")
                 .AppendLine("echo")
+                .AppendLine("oryxImageDetectorFile=\"/opt/oryx/.imagetype\"")
+                .AppendLine($"platformName=\"{platformName}\"")
+                .AppendLine($"if [ -f \"$oryxImageDetectorFile\" ] && [ \"$platformName\" = \"dotnet\" ] && grep -q \"vso-focal\" \"$oryxImageDetectorFile\"; then")
+                .AppendLine("echo \"image detector file exists, platform is dotnet..\"")
+                .AppendLine($"source /opt/tmp/build/vsoSymlinksDotNetCore.sh")
```

Comment:
Do we need these lines in this file? We've added `RUN ./opt/tmp/build/vsoSymlinksDotNetCore.sh` in `vso.focal.dockerfile`  so we don't have to source the script at here?

Reply:
no we need this script for dynamic installation scenario .. this script needs during a dynamic installation to keep the links up to date.